DocSum/kubernetes/intel/cpu/xeon/manifest/docsum.yaml

```diff
@@ -1,31 +1,4 @@
 ---
-# Source: docsum/charts/docsum-ui/templates/configmap.yaml
-# Copyright (C) 2024 Intel Corporation
-# SPDX-License-Identifier: Apache-2.0
-
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: docsum-docsum-ui-config
-  labels:
-    helm.sh/chart: docsum-ui-1.0.0
-    app.kubernetes.io/name: docsum-ui
-    app.kubernetes.io/instance: docsum
-    app.kubernetes.io/version: "v1.0"
-    app.kubernetes.io/managed-by: Helm
-data:
-  APP_BACKEND_SERVICE_ENDPOINT: "/v1/docsum"
-  APP_DATA_PREP_SERVICE_URL: "/v1/dataprep"
-  CHAT_BASE_URL: "/v1/docsum"
-  UPLOAD_FILE_BASE_URL: "/v1/dataprep"
-  GET_FILE: "/v1/dataprep/get_file"
-  DELETE_FILE: "/v1/dataprep/delete_file"
-  BASE_URL: "/v1/docsum"
-  DOC_BASE_URL: "/v1/docsum"
```

Comment:
again, [default DOC_BASE_URL](https://github.com/opea-project/GenAIExamples/blob/main/DocSum/ui/svelte/.env) value is not working in K8S environment, so we can't remove it from the config-map